src/gtfs/read.rs

```diff
@@ -267,14 +268,18 @@ impl<'de> ::serde::Deserialize<'de> for RouteType {
 impl_id!(Route);
 
 impl Route {
-    fn get_line_key(&self) -> (Option<String>, String) {
+    fn get_line_key(&self, read_as_line: bool) -> (Option<String>, String, Uuid) {
         let name = if self.short_name != "" {
             self.short_name.clone()
         } else {
             self.long_name.clone()
         };
-
-        (self.agency_id.clone(), name)
+        let key = if read_as_line {
+            Uuid::new_v4()
+        } else {
+            Uuid::NAMESPACE_OID
```

Comment:
what is this used for? it's only to always have the same value no? If so, maybe it's better to use an `Option<Uuid>` and set it to `None` no?